tests/main/parallel-install-basic/task.yaml

```diff
@@ -0,0 +1,89 @@
+summary: Checks for parallel installation of a local snap files
+
+prepare: |
+    # ensure we have no snap user data directory yet
+    rm -rf /home/test/snap
+    rm -rf /var/snap/test-snapd-tools /var/snap/test-snapd-tools_foo
+
+    snap set system experimental.parallel-instances=true
+
+execute: |
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+
+    devmode=
+    if [[ "$SPREAD_SYSTEM" = ubuntu-* ]]; then
+        # TODO prallel-install: disable devmode once AppArmor profile updates are in
```

Comment:
s/prallel/parallel/. I'm guessing the apparmor updates will be in a subsequent PR?

Reply:
Yes, as we discussed on IRC, AppAmor and interfaces is next.